sceptre-example/templates/assets-bucket.yaml

```diff
@@ -0,0 +1,14 @@
+# This is a assets bucket that we can use for All of Sceptre's templates via the
```

Comment:
I noticed that this bucket is a dependency of cdk-boostrap.yaml and ecr.yaml when running `sceptre generate ecr.yaml`  however I don't see how that relationship is setup?

Reply:
This is the template bucket that Sceptre uploads templates to before deploying via CloudFormation. See the `config/config.yaml` file.

Reply:
ahh, of course.